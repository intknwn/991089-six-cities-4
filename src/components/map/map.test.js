import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';

it(`render Map`, () => {
  const tree = renderer.create(
      <Map
        places={[]}
      />,
      {
        createNodeMock: () => document.createElement(`section`)
      }
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
