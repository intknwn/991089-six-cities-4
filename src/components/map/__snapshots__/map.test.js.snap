// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Map 1`] = `
<section
  className="cities__map map"
/>
`;
